reduce.js

```diff
@@ -32,6 +32,8 @@ function reduce(json, code) {
     }; fn`).call(json)
   }
 
+  code = createExpression(code)
```

Comment:
This will add to ALL reduce and introduce new syntax `.a-b.v-w`. Maybe try to fix it in S+> code?

Reply:
But this problem can be in any call to this function, right?